test/common_http_test.lua

```diff
@@ -0,0 +1,63 @@
+local fio = require('fio')
+local t = require('luatest')
+local g = t.group()
+
+local test_helper = require('test.helper')
+local helpers = require('cartridge.test-helpers')
+
+local http = require('http.client')
+
+g.before_all = function()
+    g.cluster = helpers.Cluster:new({
+        datadir = fio.tempdir(),
+        use_vshard = false,
+        server_command = test_helper.server_command,
+        replicasets = {{
+            alias = 'loner',
+            uuid = helpers.uuid('a'),
+            roles = {'extensions'},
+            servers = {{
+                instance_uuid = helpers.uuid('a', 'a', 1)
+            }},
+        }}
+    })
+
+    g.cluster:start()
+end
+
+g.after_all = function()
+    g.cluster:stop()
+    fio.rmtree(g.cluster.datadir)
+end
+
+local error_prefix = "Invalid extensions config: "
```

Comment:
Unused. Please pay attention to `luacheck`. There're lots of warnings across the patch.